csi/moac/test/nexus_test.js

```diff
@@ -184,10 +184,61 @@ module.exports = function() {
       callStub.reset();
     });
 
-    it('should publish the nexus', async () => {
+    it('should not publish the nexus with whatever protocol', async () => {
+      callStub.resolves({ devicePath: '/dev/whatever0' });
+      callStub.rejects(
+        new GrpcError(GrpcCode.INVALID_ARGUMENT, 'Test failure')
```

Comment:
Can we change the error code to something else (i.e. NOT_FOUND or whatever) so that we can make the distinction between the case when grpc call fails and when the argument check fails more apparent?